README.md

```diff
@@ -1,99 +1,79 @@
 
 The whole project is organized into the following directories:
 
-- `kv`: implementation of the TinyKV key/value store.
-- `proto`: all communication between nodes and processes uses Protocol Buffers over gRPC. This package contains the protocol definitions used by TinyKV, and the generated Go code that you can use.
-- `raft`: implementation of the Raft distributed consensus algorithm, which is used in TinyKV.
-- `scheduler`: implementation of the TinyScheduler which is responsible for managing TinyKV nodes and generating timestamps.
-- `log`: log utility to output log based on level.
-
-### Course material
+* `kv` contains the implementation of the key-value store.
+* `raft` contains the implementation of the Raft consensus algorithm.
+* `scheduler` contains the implementation of the TinyScheduler, which is responsible for managing TinyKV nodes and generating timestamps.
+* `proto` contains the implementation of all communication between nodes and processes uses Protocol Buffers over gRPC. This package contains the protocol definitions used by TinyKV, and the generated Go code that you can use.
+* `log` contains utility to output log based on level.
 
-Please follow the course material to learn the background knowledge and finish code step by step.
+## Build TinyKV from Source
 
-- [Project1 - StandaloneKV](doc/project1-StandaloneKV.md)
-- [Project2 - RaftKV](doc/project2-RaftKV.md)
-- [Project3 - MultiRaftKV](doc/project3-MultiRaftKV.md)
-- [Project4 - Transaction](doc/project4-Transaction.md)
+### Prerequisites
 
-## Deploy to a cluster
+* `git`: The source code of TinyKV is hosted on GitHub as a git repository. To work with git repository, please [install `git`](https://git-scm.com/downloads).
+* `go`: TinyKV is a Go project. To build TinyKV from source, please [install `go`](https://golang.org/doc/install) with version greater or equal to 1.13.
 
-After you finish the whole implementation, it becomes runnable. You can try TinyKV by deploying it onto a real cluster, and interact with it through TinySQL.
+## Setup GOPATH
 
-### Build
+If `echo $GOPATH` gives empty result, you should set up `GOPATH` firstly.
 
+```bash
+export GOPATH=$(go env GOPATH)
 ```
-make
-```
-
-It builds the binary of `tinykv-server` and `tinyscheduler-server` to `bin` dir.
 
-### Run
+## Clone
 
-Put the binary of `tinyscheduler-server`, `tinykv-server` and `tinysql-server` into a single dir.
+Clone the source code to your development machine.
 
-Under the binary dir, run the following commands:
-
-```
-mkdir -p data
+```bash
+mkdir -p $GOPATH/src/github.com/tidb-incubator
+cd $GOPATH/src/github.com/tidb-incubator
+git clone https://github.com/tidb-incubator/tinykv.git
```

Comment:
it's not necessary

Reply:
Could you elaborate it? I cannot get the point.

Reply:
```suggestion
```bash
git clone https://github.com/tidb-incubator/tinykv.git
```

I think @Cornor1996 means this, no need to put it under $GOPATH anymore.

Reply:
@you06 YES

Reply:
OK. Let me modify it.